Xiaomi/Mi10TLite-SystemUI/res/values/config.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+/*
+** Copyright 2009, The Android Open Source Project
+**
+** Licensed under the Apache License, Version 2.0 (the "License");
+** you may not use this file except in compliance with the License.
+** You may obtain a copy of the License at
+**
+**     http://www.apache.org/licenses/LICENSE-2.0
+**
+** Unless required by applicable law or agreed to in writing, software
+** distributed under the License is distributed on an "AS IS" BASIS,
+** WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+** See the License for the specific language governing permissions and
+** limitations under the License.
+*/
+-->
+<resources>
+    <!-- Default paddings for content around the corners. -->
+    <dimen name="rounded_corner_content_padding">21px</dimen>
+
+    <!-- Location on the screen of the center of the fingerprint sensor. For devices with under
+        display fingerprint sensors, this directly corresponds to the fingerprint sensor's location.
+        For devices with sensors on the back of the device, this corresponds to the location on the
+        screen directly in front of the sensor.
+        By default, this is set to @null to use the horizontal center of the screen. -->
+    <dimen name="physical_fingerprint_sensor_center_screen_location_x">1080px</dimen>
+
+    <!--  Location on the screen of the center of the fingerprint sensor. For devices with under
+         display fingerprint sensors, this directly corresponds to the fingerprint sensor's location.
+         For devices with sensors on the back of the device, this corresponds to the location on the
+         screen directly in front of the sensor.  -->
+    <dimen name="physical_power_button_center_screen_location_y">1000px</dimen>
+
+    <!-- Location on the screen of the center of the fingerprint sensor. For devices with under
+        display fingerprint sensors, this directly corresponds to the fingerprint sensor's location.
+        For devices with sensors on the back of the device, this corresponds to the location on the
+        screen directly in front of the sensor. -->
+    <dimen name="physical_fingerprint_sensor_center_screen_location_y">@dimen/physical_power_button_center_screen_location_y</dimen>
```

Comment:
your fingerprint sensor is at the same place as your power button?!? got a photo? that sounds totally messed up

Reply:
side-mounted fingerprint sensor, the sensor is in the power button

https://www.smartprix.com/bytes/xiaomi-mi-10i-5g-review-pros-cons/

![image](https://user-images.githubusercontent.com/6631572/190945100-15b94b9a-2924-4cf0-83d0-0287c80ed194.png)